extensions/amp-subscriptions/0.1/crypto-handler.js

```diff
@@ -64,17 +64,25 @@ export class CryptoHandler {
   }
 
   /**
+   * Gets encrypted document key in Base64 URL format.
+   * Returns `null` if the key wasn't found.
    * @param {string} serviceId Who you want to decrypt the key.
    *                           For example: 'google.com'
    * @return {?string}
    */
   getEncryptedDocumentKey(serviceId) {
     // Doing this for testing.
     const encryptedKeys = this.getEncryptedKeys();
-    if (!encryptedKeys) {
+    const key = encryptedKeys && encryptedKeys[serviceId];
```

Comment:
Wait. Didn't we say we won't do this? And just use a normal url-encoding instead?

Reply:
Thanks for clarifying! I realize this PR is redundant, I'll close